frontend/src/store/actions/index.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable prettier/prettier */
+import axios from 'axios';
+export const REGISTER_USER = 'REGISTER_USER';
+export const LOGIN_USER = 'LOGIN_USER';
+export const LOGOUT_USER = 'LOGOUT_USER';
+export const ADD_ARTICLE = 'ADD_ARTICLE';
+export const DELETE_ARTICLE = 'DELETE_ARTICLE';
+export const FETCH_DATA = 'FETCH_DATA';
+export const DATA_FETCHED = 'DATA_FETCHED';
+export const ERROR = 'ERROR';
+
+export const registerUser = user => {
+	return dispatch => {
+		dispatch({ type: FETCH_DATA });
+		axios
+			.post('https://anywhere-reader-test.herokuapp.com/api/users', user)
```

Comment:
To make things more manageable, take the base of the API URL `https://anywhere-reader-test.herokuapp.com/api` and make a variable of it at the top of the actions file. Then you can just concatenate the required API call bit to it in each action function, making things easier to update if we should change the URL or such.